problemsets-mat223/linearalbegra.tex

```diff
@@ -5377,13 +5377,56 @@ \section*{Eigenvectors}
 	\question
 	Define $F=\mat{1&1\\0&1}$.
 	\begin{parts}
-		\item Is $F$ diagonalizable?  Why or why not?
+		\item Is $F$ diagonalizable? Why or why not?
+			\begin{solution}
+				No.
+
+				$\chr(F)=(1-\lambda)^2$, and so the only eigenvalue of $F$ is $1$.
+				$F$ is diagonalizable if and only if there is a basis of $\R^2$
+				consisting of eigenvectors of $F$, which are vectors in the 
+				eigenspace of $F$ corresponding to its one eigenvalue. We check 
+				that 
+				\[
+					\null(F-1I)
+					=\null\left(\mat{0&1\\0&0}\right)
+					=\Span\Set*{\mat{1\\0}},
+				\]
+				which is one-dimensional, and so there is no basis of $\R^2$ 
+				consisting of eigenvectors of $F$. 
+			\end{solution}
 		\item What is the geometric and algebraic multiplicity of each eigenvalue
 			of $F$?
+			\begin{solution}
+				The only eigenvalue is $1$. Its geometric multiplicity is $1$, 
+				and its algebraic multiplicity is $2$. 
+			\end{solution}
 		\item Suppose $A$ is a matrix where the geometric multiplicity of one of its eigenvalues
-			is smaller than the algebraic multiplicity of the same eigenvalue.  Is
-			$A$ diagonalizable?  What if all the geometric and algebraic multiplicities
+			is smaller than the algebraic multiplicity of the same eigenvalue. Is
+			$A$ diagonalizable? What if all the geometric and algebraic multiplicities
 			match?
+			\begin{solution}
+				If one of the geometric multiplicities is smaller than the corresponding
+				algebraic multiplicity, $A$ cannot be diagonalizable. If $A$ is 
+				$n\times n$, then the sum of the algebraic multiplicities of the
+				eigenvalues is at most $n$. If one of the geometric multiplicities
+				is less than the corresponding algebraic multiplicity, then 
+				the sum of the dimensions of the eigenspaces must be strictly
+				less than $n$, meaning there can be no basis for $\R^n$ consisting
+				of eigenvectors. 
+
+				If all the geometric and algebraic multiplicities match, the 
```

Comment:
Change to, "Over the complex numbers, the sum of the algebraic multiplicites of an `n x n` matrix is always `n`. ..."